launch_testing/launch_testing/launch_test.py

```diff
@@ -56,7 +56,10 @@ def add_arguments(parser):
         '--junit-xml', action='store', dest='xmlpath', default=None,
         help='Do write xUnit reports to specified path.'
     )
-
+    parser.add_argument(
+        '--timeout', type=float, default=15.0,
+        help='timeout for ReadyToTest action'
```

Comment:
I think we should clarify a few things in the help message:
- this is the duration between the start of the launch process and the ReadyToTest action being executed.
- what happens if the timeout occurs
- the units are seconds